.php_cs.dist

```diff
@@ -10,6 +10,7 @@ return PhpCsFixer\Config::create()
   ->setRules([
     '@PSR2' => true,
     'array_syntax' => ['syntax' => 'short'],
+    'binary_operator_spaces' => ['operators' => ['|' => 'no_space']],
```

Comment:
Any reason for the `no_space`?
As far as I know, we should use spaces, isn't it?

![image](https://user-images.githubusercontent.com/6381924/116614401-2d693900-a93a-11eb-8aba-00488474f7cb.png)
_src: https://www.php-fig.org/psr/psr-12/_

Reply:
Totally right. Actually, this change doesn't belong to this PR at all. I will undo this change.